docs/sql/ddl.txt

```diff
@@ -449,6 +373,93 @@ Creating a partitioned table is done using the :ref:`partitioned_by_clause`::
 
```

Comment:
I find that quite misleading (for a Crate newcomer) to present a table partitioned by timestamp.

The first time I saw that (I knew nothing about shards): I thought "Great, I don't have to worry about partitions: Crate will handle my datetime partitionning automagically"

Is date a day? If yes, the granularity is far too high for most use cases.
A partition by month would be more sensible and would lead newcomers on the right track.